Makefile

```diff
@@ -257,31 +358,117 @@ changelog-unreleased: ## show the current changelog that will be produced on the
 			-t 748.5989 \
 			/CHANGELOG.md
 
-.PHONY: release
-release: clean-dist ci-bootstrap-mac changelog-release ## Build and publish final binaries and packages. Intended to be run only on macOS.
-	$(call title,Publishing release artifacts)
-
-	# Prepare for macOS-specific signing process
-	.github/scripts/mac-prepare-for-signing.sh
-
-	# create a config with the dist dir overridden
-	echo "dist: $(DISTDIR)" > $(TEMPDIR)/goreleaser.yaml
-	cat .goreleaser.yaml >> $(TEMPDIR)/goreleaser.yaml
-
-	# release (note the version transformation from v0.7.0 --> 0.7.0)
-	bash -c "\
-		BUILD_GIT_TREE_STATE=$(GITTREESTATE) \
-		VERSION=$(VERSION:v%=%) \
-		$(TEMPDIR)/goreleaser \
-			--rm-dist \
-			--config $(TEMPDIR)/goreleaser.yaml \
-			--release-notes <(cat CHANGELOG.md)"
-
-	# verify checksum signatures
-	.github/scripts/verify-signature.sh "$(DISTDIR)"
+.PHONY: homebrew-formula-generate
+.SILENT: homebrew-formula-generate
+homebrew-formula-generate:
+	$(call title,Generating homebrew formula)
+	# dependencies: curl, jq, openssl
+
+	RELEASE_URL="https://api.github.com/repos/anchore/$(BIN)/releases/tags/$(VERSION_TAG)" && \
+	echo "Using release: $${RELEASE_URL}" && \
+	curl -sSL "$${RELEASE_URL}" > "$(TEMPDIR)/release.json"
+
+	export DARWIN_AMD64_ASSET_URL=$(call asset_url,"$(TEMPDIR)/release.json","darwin_amd64.zip") && \
+	curl -sSL "$${DARWIN_AMD64_ASSET_URL}" > "$(TEMPDIR)/darwin_amd64_asset" && \
+	export DARWIN_AMD64_ASSET_SHA256=$(call sha256,"$(TEMPDIR)/darwin_amd64_asset") && \
+	\
+	export LINUX_AMD64_ASSET_URL=$(call asset_url,"$(TEMPDIR)/release.json","linux_amd64.tar.gz") && \
+	curl -sSL "$${LINUX_AMD64_ASSET_URL}" > "$(TEMPDIR)/linux_amd64_asset" && \
+	export LINUX_AMD64_ASSET_SHA256=$(call sha256,"$(TEMPDIR)/linux_amd64_asset") && \
+	\
+	export VERSION=$(call get_version_from_version_tag,$(VERSION_TAG)) && \
+	\
+	cat "./.homebrew-formula-template.rb" | \
+		envsubst > "$(HOMEBREW_FORMULA_FILE)"
+
+	echo "Generated $(HOMEBREW_FORMULA_FILE):" && \
+	cat $(HOMEBREW_FORMULA_FILE)
+
+.PHONY: homebrew-formula-test
+.SILENT: homebrew-formula-test
+homebrew-formula-test: bootstrap
+	$(call title,Testing homebrew formula)
+
+	echo "Cleaning up any versions of $(BIN) previously installed by $(BREW_CMD)"
+	$(BREW_CMD) uninstall --force "$(HOMEBREW_FORMULA_FILE)"
+
+	echo "Testing homebrew installation using formula"
+	$(BREW_CMD) install --formula "$(HOMEBREW_FORMULA_FILE)"
+
+	INSTALLED_BIN="$(BREW_BIN_DIR)/$(BIN)" && \
+	echo "Now running '$${INSTALLED_BIN} version':" && \
+	"$${INSTALLED_BIN}" version
+
+.PHONY: homebrew-formula-publish
+.SILENT: homebrew-formula-publish
+homebrew-formula-publish:
+	$(call title,Publishing homebrew formula)
+
+	FORMULA_FILE="$$(realpath $(HOMEBREW_FORMULA_FILE))" && \
+	\
+	pushd "$(TEMPDIR)" && \
+		rm -rfv "./homebrew-syft" && \
+		gh repo clone anchore/homebrew-syft && \
```

Comment:
where is `gh` coming from?

Reply:
Natively on the runners (see [this comment](https://github.com/anchore/syft/pull/310#discussion_r560967271))